mobile/src/core/interfaces.ts

```diff
@@ -1,16 +1,52 @@
 import { AuthenticateConfig } from 'react-native-touch-id';
+import { AnyAction as ReduxAction } from 'redux';
```

Comment:
Not sure if a centralized interface file is a good idea. As long as it does not grow too much it's ok to let it in a single file and later could be extracted.

Reply:
It's my thoughts exactly :)